.github/workflows/manubot.yaml

```diff
@@ -89,7 +89,7 @@ jobs:
       - name: Build Manuscript
         run: bash build/build.sh
       - name: Upload Artifacts
-        uses: actions/upload-artifact@v1
+        uses: actions/upload-artifact@v3
```

Comment:
I don't see artifacts showing up in GitHub actions anymore. Hoped this would fix it but didn't seem to.

Reply:
They are still there.  From https://github.com/manubot/rootstock/actions you can pick a run, such as https://github.com/manubot/rootstock/actions/runs/3451537811.  Then the artifacts are at the bottom.  I believe this navigation changed at some point from how you used to find the artifacts.

Reply:
Ah yes, have to go to "Summary".